packages/ui/client/composables/client/index.ts

```diff
@@ -40,9 +40,19 @@ export const client = (function createVitestClient() {
   }
 })()
 
+function sort(a: File, b: File) {
+  if (a.name < b.name)
```

Comment:
Not sure that this is correct. What if it has unicode characters? Shouldn't `.localeCompare` be used here?

Reply:
Yeah, it looks better using `localeCompare`, I've updated it.